src/main/java/com/iridium/iridiumskyblock/database/Island.java

```diff
@@ -132,11 +134,19 @@ public void setColor(Color color) {
     @Override
     public @NotNull String getName() {
         if (super.getName() != null) return super.getName();
-        String ownerName = IridiumSkyblock.getInstance().getTeamManager().getTeamMembers(this).stream()
-                .filter(user -> user.getUserRank() == Rank.OWNER.getId())
-                .findFirst()
+        String ownerName = getOwner()
                 .map(User::getName)
                 .orElse("N/A");
```

Comment:
Not sure about 4.0, but in 3.0 this text was a config option. Do we still have it?

Reply:
I have no clue, but it's out of scope of this PR since I didn't add it. If it's a config peaches or someone else should fix it

Reply:
For reference N/A is used in 11 places in V4... should have a separe PR to remove the N/A strings